backend/entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
A bit jank I feel. But it works /shrug.

Reply:
yea, but I didn't think there was another way to load the database csvs and app.db any other way.